File: ver-ios/src/budget/budgetEffect.swift
import gb
import UIKit

func budgetCopyResult(_ result: String) {
    UIPasteboard.general.string = result
}

func budgetPasteMorningBalance() {
    let txt = UIPasteboard.general.string ?? ""
    budgetSet(F.pastedMorningBalance, txt)
}

func budgetPasteSpent() {
    let txt = UIPasteboard.general.string ?? ""
    budgetSet(F.pastedSpent, txt)
}

File: ver-mac-x64/src/budget/budgetEffect.cpp
#include <QClipboard>
#include <QGuiApplication>

#include "budgetEffect.h"
#include "ignore.kd.h"

void budgetCopyResult(const QString &r) {
    QGuiApplication::clipboard()->setText(r);
}

void budgetPasteMorningBalance() {
    QString txt = QGuiApplication::clipboard()->text();
    budgetSet(F.pastedMorningBalance, txt.toStdString().c_str());
}

void budgetPasteSpent() {
    QString txt = QGuiApplication::clipboard()->text();
    budgetSet(F.pastedSpent, txt.toStdString().c_str());
}
